etc/PrimarySourceSetDataModelSample.json

```diff
@@ -37,7 +37,7 @@
       "name": "Thirteen Colonies"
     }
   ],
-  "dplaer:timePeriod": [
+  "dct:temporal": [
     { "name": "Colonial America" }
```

Comment:
Can we add a statement here asserting `@type: dct:PeriodOfTime`?  Also, I just want to confirm that `dct:PeriodOfTime` is expected to have a `name` property.